frontend/src/lang/es.ts

```diff
@@ -195,6 +195,7 @@ export const es: LanguageConfig = {
           street: "Falta el nombre de la calle",
           streetTwo: "Calle 2 tiene un formato incorrecto",
           zipCode: "Falta el código postal o tiene un formato incorrecto",
+          zipForState: "Código postal no válido para este estado",
```

Comment:
@Rebecca-LM can you please review this translation?

Reply:
@mehansen I'd change to: "El código postal no es válido para este estado"